test/data/html/tests.js

```diff
@@ -684,8 +684,8 @@ exports.test_data = {
                 output: '<div {{#if test}} class="foo" {{/if}}>^^^&content$$$</div>'
             }, {
                 fragment: true,
-                input_: '<div{{#if thing}}{{somestyle}}class="{{class}}"{{else}}class="{{class2}}"{{/if}}>^^^&content$$$</div>',
-                output: '<div {{#if thing}} {{somestyle}} class="{{class}}" {{else}} class="{{class2}}" {{/if}}>^^^&content$$$</div>'
+                input_: '<div{{#if thing}}{{somestyle}}class="{{c1}}"{{else}}class="{{c2}}"{{/if}}>^^^&content$$$</div>',
```

Comment:
Rather than replacing the existing test with a different one, keep the existing test(s) and then add the new test(s).